packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -130,18 +172,20 @@ export const SelectControl = ( {
 			<label { ...getLabelProps() }>{ label }</label>
 			{ /* eslint-enable jsx-a11y/label-has-for */ }
 			<div className="woocommerce-experimental-select-control__combo-box-wrapper">
-				{ multiple && (
+				{ multiple && multipleSelection && (
```

Comment:
Any particular purpose for not destructuring props from `multipleSelection`?

The logic around this becomes a little more confusing IMO by having multiple checks for properties that should inherently exist.

Reply:
No particular reason, I think previously I was running into some naming overlap, so not de-structuring prevented that. But I should be able to change that back.